pkg/api/accesscontrol.go

```diff
@@ -458,7 +458,7 @@ func (hs *HTTPServer) declareFixedRoles() error {
 	generalFolderReaderRole := ac.RoleRegistration{
 		Role: ac.RoleDTO{
 			Name:        "fixed:folders.general:reader",
-			DisplayName: "General folder reader",
```

Comment:
I think this one is not needed. This is Internal i would guess? I want to investigate more on what this roles actually gives you..

Reply:
It's only the general uid folder so this makes sense 🚀

Reply:
This gives you write access to the Folder General, this folder sits at the root of your grafana and grants you write permission on all child folders.

We tend to call it the General folder and not the root folder but naming varies 🤷 (if you create a new dashboard and try to save it, it will have the name "Dashboards")

Reply:
I did notice a couple places with `general` and `root` seemingly meaning the same thing... now would be a good time to standardize if we want to since I'm already renaming a bunch of roles.  Does `general` make more sense?

Reply:
I think for now we keep it as general. Alot of workflows might depend on it. Let's rename for root when we potentially move to use more of the workspace approach that Sarah showed

Reply:
Sounds good, I'll change it back.